### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henryosei**
Updated the readme file to add excel parser (readtafia/calamine)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1439,6 +1439,7 @@ See also [Are we game yet?](https://arewegameyet.rs)
 ### Parsing
 
   * [comex/rust-shlex](https://github.com/comex/rust-shlex) [[shlex](https://crates.io/crates/shlex)] — Split a string into shell words, like Python's shlex. [![build badge](https://github.com/comex/rust-shlex/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/comex/rust-shlex/actions/workflows/test.yml)
+  * [tafia/calamine](https://github.com/tafia/calamine) - An Excel/OpenDocument Spreadsheets file reader/deserializer, in pure Rust.
```
Comment (Israel-Laguan) [inline README.md:1442]: Please add build badge if possible

Comment (Israel-Laguan) [reply README.md:1442]: Also, seems like the order goes on the first letter, in this case the "t" in "tafia", so maybe you need to move this item below

Comment (henryosei) [reply README.md:1442]: Making the changes

**Comment by Israel-Laguan** [comment]
Found 1.1k+ stars
image

**Review by Israel-Laguan — APPROVED** [review]

**Pull request closed without merging**